### PR TITLE
Make i5 variation default pricing page

### PR DESCRIPTION
**Pull request by monsieur-z**
### Changes proposed in this Pull Request

This PR makes the i5 variation of the pricing page the default page.

### Implementation notes

Changelog:
- Moved and renamed `JetpackFreeCard` to `client/components/jetpack/card/jetpack-free-card/`
- Moved and renamed `JetpackProductCard` to `client/components/jetpack/card/jetpack-product-card/`
- Removed unused `jetpack-product-slide-out-card` component
- Removed i5 specific CSS selectors
- Moved and renamed `JetpackFAQ` to `client/my-sites/plans/jetpack-plans/faq/`
- Made i5 copy default
- Removed declared iterations
- Moved and renamed `PlansFilterBar` to `client/my-sites/plans/jetpack-plans/plans-filter-bar/`
- Moved and renamed `ProductCard` to `client/my-sites/plans/jetpack-plans/product-card/`
- Moved and renamed `ProductGrid` to `client/my-sites/plans/jetpack-plans/product-grid/`

### Testing instructions

Download the PR and run both cloud or Calypso concurrently.

For each platform:
- Visit the pricing page locally
- Open a new tab and visit the pricing page in staging or production
- Check that both pages look and behave similarly
- Verify there's no error in the console

### CODE REVIEW

File: client/my-sites/plans/jetpack-plans/constants.ts [hunk 1]
```diff
@@ -385,23 +376,7 @@ export const EXTERNAL_PRODUCTS_SLUG_MAP: Record<
  * Constants that contain products including option and regular types.
  */
 
-export const SELECTOR_PLANS_ALT_V1 = [
```
Comment (monsieur-z) [inline client/my-sites/plans/jetpack-plans/constants.ts:392]: Unused.

File: client/my-sites/plans/jetpack-plans/constants.ts [hunk 2]
```diff
@@ -385,23 +376,7 @@ export const EXTERNAL_PRODUCTS_SLUG_MAP: Record<
...
-	PLAN_JETPACK_SECURITY_DAILY,
-	PLAN_JETPACK_SECURITY_DAILY_MONTHLY,
-	PLAN_JETPACK_COMPLETE,
-	PLAN_JETPACK_COMPLETE_MONTHLY,
-];
-
-export const SELECTOR_PLANS_ALT_V2 = [
```
Comment (monsieur-z) [inline client/my-sites/plans/jetpack-plans/constants.ts:399]: Unused.

File: client/my-sites/plans/jetpack-plans/constants.ts [hunk 3]
```diff
@@ -410,26 +385,6 @@ export const SELECTOR_PLANS_I5 = [
 	PLAN_JETPACK_COMPLETE_MONTHLY,
 ];
 
-export const FEATURE_TO_PRODUCT_ALT_V2: Record< string, string > = {
```
Comment (monsieur-z) [inline client/my-sites/plans/jetpack-plans/constants.ts:417]: Unused.

File: client/my-sites/plans/jetpack-plans/constants.ts [hunk 4]
```diff
@@ -410,26 +385,6 @@ export const SELECTOR_PLANS_I5 = [
...
-	[ FEATURE_PRODUCT_BACKUP_DAILY_V2 ]: PRODUCT_JETPACK_BACKUP_DAILY,
-	[ FEATURE_PRODUCT_BACKUP_REALTIME_V2 ]: PRODUCT_JETPACK_BACKUP_REALTIME,
-	[ FEATURE_PRODUCT_SCAN_V2 ]: PRODUCT_JETPACK_SCAN,
-	[ FEATURE_PRODUCT_ANTISPAM_V2 ]: PRODUCT_JETPACK_ANTI_SPAM,
-	[ FEATURE_SEARCH_V2 ]: PRODUCT_JETPACK_SEARCH,
-	[ FEATURE_PRODUCT_SEARCH_V2 ]: PRODUCT_JETPACK_SEARCH,
-	[ FEATURE_CRM_V2 ]: PRODUCT_JETPACK_CRM,
-};
-
-export const FEATURE_TO_MONTHLY_PRODUCT_ALT_V2: Record< string, string > = {
```
Comment (monsieur-z) [inline client/my-sites/plans/jetpack-plans/constants.ts:427]: Unused.

**Review by rcanepa — APPROVED** [review]
LGTM.

Changes are definitely sound and all plans grids tested well. Amazing work!

I wonder if these changes could introduce some friction to the work @mattgawarecki has been doing regarding the new iteration. Wouldn't merging this PR before Matt's work make a rebase/merge for him much difficult?

**Comment by monsieur-z** [comment]
> Wouldn't merging this PR before Matt's work make a rebase/merge for him much difficult?

Definitely. I was planning to merge it once Matt has deployed the latest experiment. What's your take on this @mattgawarecki ?

**Comment by mattgawarecki** [comment]
@monsieur-z I'm launching the experiment now. Though there will definitely be some merge conflicts, I think we'll be able to sort them out pretty quickly. More than happy to help if you'd like 🙂

**Comment by monsieur-z** [comment]
Since rebasing wasn't trivial, could I get a new review for this PR?

(Note that it nows replaces _Jetpack Complete_ by _Complete_)

**Review by mattgawarecki — APPROVED** [review]
Looking good! I tinkered with pricing pages in both Calypso Blue and Green, clicked buttons, toggled billing terms, and checked the console for errors. Everything works as expected, and we've got a lot less extra code hanging around.

Big 👍  from me!

**Pull request merged**